src/main/validation/PromotionValidation.ts

```diff
@@ -43,6 +43,7 @@ export class PromotionValidation {
       .default(new Date()),
     /** note incoming format is a string, and Joi will automatically convert to type Date */
     expirationDate: Joi.date().required(),
+    googlePlacesAddress: Joi.string().required(),
```

Comment:
How about `address`? That way, we're not coupling it to the implementation details / choice of library